README.md

```diff
@@ -19,5 +19,34 @@ The repository can be found [here](https://github.com/nl-portal/nl-portal-backen
 
 There is a React and Angular frontend version available. The frontend development of the React version can be found [here](https://github.com/nl-portal/nl-portal-frontend-libraries). More information about NL design can be found on https://nldesignsystem.nl/. 
 
+## Designs
+
+UI Designs are made in Figma and can be found here: (Instructions in Dutch)
```

Comment:
Waarom in Nederlands? Kunnen we ze niet beter vertalen om in lijn met de rest te zijn?

Reply:
Je hebt  gelijk. Het was een Copy-paste. Nu gefixed